cloud/amazon/ec2.py

```diff
@@ -613,6 +620,19 @@ def get_instance_info(inst):
         instance_info['ebs_optimized'] = False
 
     try:
+        bdm_dict = {}
```

Comment:
What do these lines have to do with the described new features? This seems to be bleed-over from another feature you're working on?